TRB_label_assist/models.py

```diff
@@ -315,17 +316,23 @@ def fit(self, train_df):
         if len(self.train_df) == 0:
             # i.e. no valid trips after removing all nans
             raise Exception('no valid trips; nothing to fit')
-
+        
+        model_config = {
```

Comment:
The changes her, from line 319 till 335 are the same changes as clustering.py file. They remove dependency from `eamts` in `tour_model_extended` in custom branch.